src/main/kotlin/com/michaelsgroi/warrencromartie/War.kt

```diff
@@ -1,10 +1,15 @@
-package com.michaelsgroi.baseballreference
+package com.michaelsgroi.warrencromartie
 
+import com.michaelsgroi.baseballreference.BrWarDailyLines
+import com.michaelsgroi.baseballreference.Career
+import com.michaelsgroi.baseballreference.Roster
 import com.michaelsgroi.baseballreference.Roster.RosterId
+import com.michaelsgroi.baseballreference.Season
+import com.michaelsgroi.baseballreference.SeasonLine
 import java.math.RoundingMode.HALF_UP
 import kotlin.math.roundToInt
 
-class BrWarDaily {
+class War {
```

Comment:
I appreciate simplifying the name `BrWarDaily`; however, this seems potentially confusing—at least I imagine "WAR" to refer to a numerical score (representing, of course, the sabermetric concept of wins above replacement), whereas, well, that's not wat instances of this class `War` represent (and if I see a `War` object `war` in code, I'd expect the former, not this...) 🤷 if the goal here is merely simplifying the name (a good goal), then I suggest... hmm... naming is hard

Reply:
Yeah, I need a name for the abstraction over these two data sources ... a name that I don't have.  I suppose I could just call it `WarrenCromartie`.

Reply:
WarSource?